bionemo-recipes/recipes/esm2_native_te/hydra_config/defaults.yaml

```diff
@@ -16,6 +16,7 @@ dataset:
   num_workers: 1
   max_seq_length: 1024
   use_sequence_packing: false
+  use_token_packing: false
```

Comment:
why not make this true? It seems like what a user would _usually_ want unless they're doing some kind of test.

Reply:
oh this is because you are doing BSHD for this example?

Reply:
Yeah, I think we can probably deprecate the BSHD-compatible use case outside of our test suite, and just have 2 dataloaders:
1. BSHD for compatibility with the HF checkpoints
2. THD where we pack to some maximum token number